vtex-sites/storecomponents.store
# Decrease bundle sizes

## What's the purpose of this pull request?
Upgrades to latest version of faststore that fixes some gatsby and react-intl bloatware.

## How it works? 
More info on https://github.com/vtex/faststore/pull/699

## How to test it?
Open /bundle-stats.html and compare from the one in master. You may see a great reduction in sizes
